### PR TITLE
Fixed PHP fatal error in gallerya template override filter.

### PR DESCRIPTION
**Pull request by juanlopez4691**
### Ticket
- [PHP Fatal error on plugin gallerya](https://app.asana.com/0/383242129844224/1112903661454517)

### CODE REVIEW

File: src/Plugin.php [hunk 1]
```diff
@@ -75,12 +75,17 @@ public static function wp_enqueue_scripts() {
    * @implements post_gallery
    */
   public static function post_gallery($output = '', $atts) {
+    if (empty($atts)) {
```
Comment (fabianmarz) [inline src/Plugin.php]: Please read [this section](http://php.net/manual/en/functions.arguments.php#functions.arguments.default) about function default arguments…

You should use `$atts = []` in the functions parameter list, than checking for empty.

Comment (juanlopez4691) [reply src/Plugin.php]: @fabianmarz I know about default arguments, sure 😄 . The problem is that the argument `$atts` is not undefined but set as `` (empty string), so the default value would not be applied. That's why I am checking for empty and then setting $args as an empty array if necessary.

Comment (juanlopez4691) [reply src/Plugin.php]: If you prefer, I can do:
```php
public static function post_gallery($output = '', $atts = []) {
    if (!is_array($atts)) {
      $atts = [];
    }
...
```
This way not having a default value for the last argument in the call (phpcs complains) is also fixed.

**Comment by fabianmarz** [comment]
@juanlopez4691, I added a commit which fixes the issue and makes the code more readable.

I wonder, how the images are displayed correctly as the `post__in` query var expects an array, not a string 🤔 I changed it to be the `include` query var as it's used by the default `post_gallery` shortcode as well.

**Comment by juanlopez4691** [comment]
> @juanlopez4691, I added a commit which fixes the issue and makes the code more readable.
> 
> I wonder, how the images are displayed correctly as the `post__in` query var expects an array, not a string 🤔 I changed it to be the `include` query var as it's used by the default `post_gallery` shortcode as well.

Your solution looks beautiful! But now we need someone else to approve the PR 😆 I will pass it to @LucaPipolo.

**Comment by juanlopez4691** [comment]
Wrong name 🙈 I meant @fabianmarz

**Pull request merged**